README.md

```diff
@@ -25,6 +25,8 @@ being too spammy in text channels.**
 - `!help`: Provides detailed help about all of the following commands.
 - `!about`: Get information about SpellBot and its creators.
 
+To use the [commands for event runners](#%EF%B8%8F-commands-for-event-runners), you will need to [create the roll on your server][create-role] called `SpellBot Admin` (capitalization matters). Only users on your server that have that role will be able to use those commands.
```

Comment:
Whoops. Roll -> role

Reply:
Hehe I noticed that after the fact too. Fixed it in https://github.com/lexicalunit/spellbot/commit/6b552880d57caf84bd1cd74ff478afd7ee0b5f4f